### DISCUSSION TITLE
MSCS%40UCSB/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#245] [Announcements]
# MSCS%40UCSB/

None

https://opencs.app/MSCS%40UCSB/

**Comment by HUST-AI-HYZ** [comment]
个人觉得不应该在A-， 应该在A一档

**Reply by HUST-AI-HYZ** [reply]
24fall 听说国际生录取比例只有2%，19/989的录取比例，被录取的dp了解到有两财一贸cs本科，个人感觉这个项目太过于random以至于申请价值不高。
